script.js

```diff
@@ -338,6 +338,102 @@ addClickEventListener(
   syncNicknamesStatusUpdate,
   'POST',
 );
+
+const DROPDOWN_OPTIONS = [
+  {
+    name: 'Home',
+    link: 'https://dashboard.realdevsquad.com/',
+  },
+  {
+    name: 'Status',
+    link: 'https://my.realdevsquad.com/',
+  },
+  {
+    name: 'Profile',
+    link: 'https://my.realdevsquad.com/profile',
+  },
+  {
+    name: 'Tasks',
+    link: 'https://my.realdevsquad.com/tasks',
+  },
+  {
+    name: 'Identity',
+    link: 'https://my.realdevsquad.com/identity',
+  },
+];
+
+async function handleUserSignin() {
+  try {
+    const self_user = await getSelfUser();
+    if (self_user) {
+      const signInButton = document.querySelector('.sign-in-btn');
+      signInButton.style.display = 'none';
+      const dropdown = document.getElementById('dropdown');
+      const userInfo = document.querySelector('.user-info');
+      const username = document.getElementById('user-name');
+      const userImage = document.getElementById('user-img');
+
+      username.innerText = `Hello, ${self_user.first_name}!`;
+      userImage.setAttribute('src', self_user?.picture.url);
+      userInfo.classList.add('active');
+      const dropdownList = createElement({
+        type: 'ul',
+        attributes: {
+          class: 'dropdown-list',
+        },
+      });
+
+      DROPDOWN_OPTIONS.map((option) => {
```

Comment:
what's the reason for using map here?

Reply:
Nothing specific. should i change it to forEach ?